src/game/gui/gauge.c

```diff
@@ -4,6 +4,7 @@
 #include "utils/miscmath.h"
 #include "video/surface.h"
 #include "video/video.h"
+#include "formats/palette.h"
```

Comment:
Is this used ?

Reply:
Palette has the definition for transparency indices.